EvidenceApi.Tests/V1/E2ETests/DocumentSubmissionsTest.cs

```diff
@@ -121,6 +121,37 @@ public async Task CanUpdateDocumentSubmissionStateWithValidParameters()
             result.Should().BeEquivalentTo(expected);
         }
 
+        [Test]
+        public async Task CanUpdateDocumentSubmissionVisibilityWithValidParameters()
```

Comment:
Would be good to add tests for both hiding and unhiding the document.

Reply:
Addressed in [9ca749b](https://github.com/LBHackney-IT/evidence-api/pull/235/commits/9ca749beadbdee4cf7c7e25055407fe060070e52)